src/commonTest/kotlin/kotools/types/number/StrictlyNegativeIntTest.kt

```diff
@@ -71,6 +72,39 @@ class StrictlyNegativeIntTest {
             .shouldHaveAMessage()
     }
 
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrNull_should_not_be_null_with_a_strictly_negative_Int() {
+        val number: Number = StrictlyNegativeInt.random().toInt()
+        val result: StrictlyNegativeInt? = number.toStrictlyNegativeIntOrNull()
+        result?.toInt() shouldEqual number
+    }
+
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrNull_should_be_null_with_a_positive_Int() {
+        val number: Number = PositiveInt.random().toInt()
+        val result: StrictlyNegativeInt? = number.toStrictlyNegativeIntOrNull()
+        assertNull(result)
+    }
+
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrThrow_should_pass_with_a_negative_Int() {
+        val number: Number = StrictlyNegativeInt.random().toInt()
+        number.toStrictlyNegativeIntOrThrow().toInt() shouldEqual number
+    }
+
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrThrow_should_throw_with_a_positive_Int() {
+        val number: Number = PositiveInt.random().toInt()
+        val exception: IllegalArgumentException = assertFailsWith<IllegalArgumentException> {
+            number.toStrictlyNegativeIntOrThrow()
+        }
+        exception.shouldHaveAMessage()
```

Comment:
We should align assertions by using the [`shouldFailWithIllegalArgumentException`](https://github.com/kotools/types/blob/39c92ca9cd9b0373f5613b7e5da6950e278715ba/src/commonTest/kotlin/kotools/types/Assertions.kt#L24) function instead.

```suggestion
        number.shouldFailWithIllegalArgumentException {
            toStrictlyNegativeIntOrThrow()
        }.shouldHaveAMessage()
```